packages/runtime/container-runtime/src/test/opLifecycle/batchManager.spec.ts

```diff
@@ -165,73 +159,12 @@ describe("BatchManager", () => {
 	});
 
 	it("Batch content size is tracked correctly", () => {
-		const batchManager = new BatchManager({ hardLimit }, mockLogger);
+		const batchManager = new BatchManager({ hardLimit });
 		assert.equal(batchManager.push(smallMessage()), true);
 		assert.equal(batchManager.contentSizeInBytes, smallMessageSize * batchManager.length);
 		assert.equal(batchManager.push(smallMessage()), true);
 		assert.equal(batchManager.contentSizeInBytes, smallMessageSize * batchManager.length);
 		assert.equal(batchManager.push(smallMessage()), true);
 		assert.equal(batchManager.contentSizeInBytes, smallMessageSize * batchManager.length);
 	});
-
-	it("Don't verify op ordering by default", () => {
-		const batchManager = new BatchManager({ hardLimit }, mockLogger);
-		assert.equal(batchManager.push({ ...smallMessage(), referenceSequenceNumber: 0 }), true);
-		assert.equal(batchManager.push({ ...smallMessage(), referenceSequenceNumber: 0 }), true);
-		assert.equal(batchManager.push({ ...smallMessage(), referenceSequenceNumber: 1 }), true);
```

Comment:
It feels like we still need these tests, it's just that we should test for different outcome, no?

Reply:
The batch manager is no longer governing this. The tests will be added to the outbox.